docs/integrations/saas-cloud/docusign.md

```diff
@@ -0,0 +1,209 @@
+
+```json
+{
+ "timestamp": "2023-02-20T10:42:19.8996904Z",
+ "eventId": "285e7634-07fd-4877-ac12-b921d0e4b626",
+ "site": "DEMO",
+ "accountId": "",
+ "organizationId": "7f46f1ee-828e-41d4-9d85-755fa1814fd4",
+ "userId": "9cfb472b-ef1f-4116-8df2-17c53841fd6c",
+ "integratorKey": "",
+ "userAgent": "",
+ "ipAddress": "103.108.207.58",
+ "ipAddressLocation": {
+   "latitude": 23.02,
+   "longitude": 72.57,
+   "country": "india",
+   "state": "gujarat",
+   "city": "ahmedabad"
+ },
+ "object": "Alert",
+ "action": "IpAddressOutsideAllowedRanges",
+ "property": "",
+ "field": "",
+ "result": "",
+ "source": "Other",
+ "referencedUserId": "",
+ "proxyStatus": "",
+ "proxyType": "",
+ "proxyLevel": "",
+ "latitude": 23.02,
+ "longitude": 72.57,
+ "city": "ahmedabad",
+ "state": "gujarat",
+ "country": "india",
+ "data": {
+   "EventIds": [
+     "16b4a651-1e21-42a3-90ea-f5227224b591",
+     "1addc090-d227-4166-a279-dc6c5277247b",
+     "1e0f0d65-ab89-4b74-9f85-6f80b0cd59b4",
+     "205bb7fe-bada-4331-bfcb-73a81d0bebfd",
+     "260ff2b6-c818-409a-813b-91d4b5afee27",
+     "374f01a9-c86e-4600-b8e5-d20cd699fc65",
+     "3a28a286-b041-42e8-aaac-fdb8234951f1",
+     "3bc218bd-b8d6-4861-b471-f642c4b5aaca",
+     "40c2e748-bd09-45c9-958b-e89e3280f716",
+     "44badf68-6b7b-4294-b6bb-e1943dc0095b",
+     "88cd1d2f-87a5-49e5-92cf-39cdf9550be1",
+     "897bf5dc-2902-4270-a67e-dfd25fe98893",
+     "9cc9a7f1-a853-45ab-95c1-22c7e7834e1d",
+     "aaf22d31-9afa-48f3-880e-6287e4179daf",
+     "b4cc9614-6606-4fd9-86df-fad67b2f6181",
+     "de8ebe47-9e94-42e4-86db-4d56f882ab0a",
+     "e9e27037-213f-46db-9976-2b22c1b36bc9",
+     "f767e823-9722-49ef-8da5-b0d9f8dd13b6"
+   ],
+   "Count": "18"
+ }
+}
+```
+
+### Sample Alerts Query
+
+```sql title="Alerts by Severity Over Time"
+_sourceCategory="docusign_src"
+| json "object","userId","eventId","action","property","source","ipAddressLocation.latitude","ipAddressLocation.longitude","result","ipAddressLocation.city","ipAddressLocation.state","ipAddressLocation.country" as object,user_id,event_id,action,property,source,latitude,longitude,result,city,state,country nodrop
+| where object matches "{{object}}" and action matches"{{action}}" and source matches"{{source}}"
+| where country matches "{{country}}" or isNull(country)
+| where object matches("*Alert*")
+| lookup alert,object,ds_action,description,severity,severity_id from https://sumologic-app-data.s3.amazonaws.com/docusign_alert_lookup.csv on action = ds_action
+| if(isNull(severity),"low",severity) as severity
+| where severity matches "{{severity}}"
+| timeslice 1d
+| count_distinct(event_id) as frequency by _timeslice,severity
+| fillmissing timeslice,values all in severity
+| transpose row _timeslice column severity as low, medium, high
+```
+
+## Installing the DocuSign App
+
+Locate and install the app from the **App Catalog**. If you want to see a preview of the dashboards included with the app before installing, click **Preview Dashboards**.
```

Comment:
```suggestion
Locate and install the app from the **App Catalog**.  To view a preview of the dashboards included with the app before installing, click **Preview Dashboards**.
```

Reply:
I guess `To preview the dashboards included with the app before installing` would be more appropriate.